### PR TITLE
Fixes SAML and Github connectors schema

### PR DESCRIPTION
**Pull request by gzigzigzeo**
What's inside:

- [x] Changed object constructors logic
- [x] Marked required SAML attributes as computed

### CODE REVIEW

**Review by Joerger — APPROVED** [review]

File: terraform/example/oidc_connector.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Teleport OIDC connector
```
Comment (tcsc) [inline terraform/example/oidc_connector.tf:1]: Why commented out?

File: terraform/example/trusted_cluster.tf [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Teleport trusted cluster
```
Comment (tcsc) [inline terraform/example/trusted_cluster.tf:1]: Also why commented out?

**Review by tcsc — APPROVED** [review]

File: terraform/example/oidc_connector.tf (hunk 1 above)
Comment (gzigzigzeo) [reply terraform/example/oidc_connector.tf:1]: For testing purposes, uncommented it back

**Comment by gzigzigzeo** [comment]
This conflict is about `protoc-gen-terraform` version. In this branch, it points to a correct version.

**Pull request closed without merging**